pnpm-workspace.yaml

```diff
@@ -2,3 +2,4 @@ packages:
   - "packages/**"
   - "shared"
   - "!examples/next14-approuter-html-submission/*"
+  - "!examples/next15-approuter-html-submission/*"
```

Comment:
Wasn't sure why this change was needed for the next14 example because I'm not particularly familiar with pnpm, but just duplicated the line to match for next15 anyway...